code_generation/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jinja2
+dataclasses_json
```

Comment:
Nitish already add these two in the dev dependency. We don't need this file here.

Reply:
then remove the dependency in the toml